shell.nix

```diff
@@ -1,19 +1,27 @@
-{ pkgs ? import ./pkgs.nix, env ? "staging" }:
+{ pkgs ? import ./pkgs.nix
+, env ? "staging" }:
 
 with pkgs;
 
 stdenv.mkDerivation {
   name = "disciplina-nixops";
-  nativeBuildInputs = [ git-crypt nixops sqlite ];
+  nativeBuildInputs = [
+    awscli
+    git-crypt
+    nixops
+    sqlite
+  ];
 
   AWS_ACCESS_KEY_ID = "default";
   AWS_SHARED_CREDENTIALS_FILE = "${toString ./.}/keys/${env}/aws-credentials";
 
+  DISCPLINA_ENV = env; # Fancy way to share this variable with nixops
   NIX_PATH = lib.concatStringsSep ":" [
     "nixpkgs=${pkgs.path}"
     "disciplina=https://github.com/DisciplinaOU/disciplina/archive/master.tar.gz"
     "disciplina-faucet-frontend=https://github.com/DisciplinaOU/disciplina-faucet-frontend/archive/master.tar.gz"
     "disciplina-explorer-frontend=https://github.com/DisciplinaOU/disciplina-explorer-frontend/archive/master.tar.gz"
     "disciplina-validatorcv=https://github.com/DisciplinaOU/disciplina-validatorcv/archive/master.tar.gz"
+    "local=${builtins.toString ./.}"
   ];
 }
```

Comment:
I remember you and @truekubas were discussing that some environmental variable should carry disciplina backend URL so that the frontend could use it. Where is it handled?

Reply:
That is used [here](https://github.com/DisciplinaOU/disciplina-validatorcv/blob/master/release.nix), and configured [here](https://github.com/DisciplinaOU/disciplina-nixops/blob/master/deployments/cluster/balancer.nix#L79).